IoTEdgeModbus/Modules/AzureIoTEdgeModbus/Slave/ModbusComplexValuesHandler.cs

```diff
@@ -1,25 +1,21 @@
-﻿using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
-using System.Linq;
-using System.Runtime.InteropServices.ComTypes;
-using System.Text;
-
-namespace AzureIoTEdgeModbus.Slave
+﻿namespace AzureIoTEdgeModbus.Slave
```

Comment:
At some point we may need to create encoding code for writes based on the decoding pattern.

Reply:
Yes, I agree.
When write support is added, we should also add the encoder classes.